dronegen/windows.go

```diff
@@ -23,8 +23,8 @@ const (
 	toolchainDir      = `/toolchains`
 	teleportSrc       = `/go/src/github.com/gravitational/teleport`
 
-	relcliURL    = `https://cdn.teleport.dev/relcli-v1.1.76-windows.exe`
-	relcliSha256 = `56dfdd9d1a09aac892fcd48eba035072dc6c151eaa2e1b21cf54786bb3c09520`
+	relcliURL    = `https://cdn.teleport.dev/relcli-master-e148541-20230331T1403513-windows.exe`
```

Comment:
Should we be using a versioned build instead of a commit hash?

Reply:
@justinas this is THE version we use now, right? No v1.x.x?

Reply:
I merged it as is, we can revisit it later

Reply:
Sorry, I missed this conversation. We have stopped tagging vX.Y.Z in Cloud, and as relcli follows Cloud's versioning, we do not have "traditional" versions for relcli either.